run acceptance tests in zuul job

fredericlepied opened the pull request.

fredericlepied committed:
run acceptance tests in zuul job

fblackburn1 commented on zuul.d/run.yaml:
```diff
@@ -22,3 +22,37 @@
+              acceptance: True
+              linphone: False  # otherwise, there are unprintable (^H) character that jenkins cannot decode in junit.xml
+            setupd:
+              timeout: 120
```
this timeout is now useless, the default has been increased to 60 seconds (I removed it from jenkins config)

https://github.com/wazo-pbx/wazo-setupd-client/blob/master/wazo_setupd_client/commands/setup.py#L7

fredericlepied committed:
zuul.d/pre.yaml: create an ssh key to connect as root

fredericlepied committed:
zuul.yaml: wazo-pbx -> wazo-platform

fredericlepied committed:
use 127.0.0.1 instead of localhost

reason: for an unknown reason, linphone or zuul node doesn't work with
localhost. Maybe the zuul node doesn't convert localhost to 127.0.0.1
(i.e. 127.0.1.1 or ipv6 addr)
